mods/seasonal/moves.js

```diff
@@ -620,7 +626,7 @@ exports.BattleMovedex = {
 		pp: 10,
 		priority: 0,
 		category: "Special",
-		shortDesc: "For 5 turns, negates all Ground immunities. More power the heavier the target.",
+		shortDesc: "Adds Trick Room. Deals damage.",
```

Comment:
Might want to include how its weight dependent (more damage the heavier the user is than the target)

Reply:
No, short desc needs to be very short do it fits within ``/data``

Reply:
Fair enough but perhaps something like "Adds Trick Room; more power heavier the user."

Reply:
that works